channels/1ds-post-js/src/HttpManager.ts

```diff
@@ -1490,4 +1500,23 @@ export class HttpManager {
     public sendSynchronousBatch(batch: EventBatch, sendType?: EventSendType, sendReason?: SendRequestReason) {
         // @DynamicProtoStub - DO NOT add any code as this will be removed during packaging
     }
+
+    
+    /**
+     * Get Offline Serializer support
+     * @returns internal Offline Serializer object
+     */
+    public _serializeOfflineEvt(evt: ITelemetryItem | IPostTransmissionTelemetryItem): string {
```

Comment:
minor nit: public functions should not "really" begin with an "_"

We can (and do) do this for internal only functions. But as the `HttpManager` is all internal anyway we don't need to to that here.